exchanges/btse/btse_wrapper.go

```diff
@@ -875,11 +875,18 @@ func (b *BTSE) GetOrderHistory(ctx context.Context, getOrdersRequest *order.GetO
 			if !matchType(currentOrder[y].OrderType, orderDeref.Type) {
 				continue
 			}
+			orderTime := time.Unix(currentOrder[y].Timestamp, 0)
```

Comment:
from the docs it seems that the timestamp is in milliseconds, if I am not mistaken, we can probably change this to:
`orderTime := time.Unix(0, convert.UnixMillisToNano(currentOrder[y].Timestamp))`